example-todomvc/src/main/scala/mhtml/todo/MhtmlTodo.scala

```diff
@@ -0,0 +1,342 @@
+// This implementation is mostly copied from Binding.scala TodoMVC example:
+// https://github.com/ThoughtWorksInc/todo/blob/master/js/src/main/scala/com/thoughtworks/todo/Main.scala
+package mhtml.todo
+
+import scala.scalajs.js.JSApp
+import scala.xml.{Elem, Node, Null, UnprefixedAttribute}
+import scala.collection.breakOut
+import cats._
+import cats.data._
+import cats.implicits._
+import mhtml._
+import mhtml.implicits.cats._
+import org.scalajs.dom
+import org.scalajs.dom.{Event, HashChangeEvent, KeyboardEvent}
+import org.scalajs.dom.ext.KeyCode
+import org.scalajs.dom.ext.LocalStorage
+import org.scalajs.dom.raw.HTMLInputElement
+import upickle.default.read
+import upickle.default.write
+
+import scala.scalajs.runtime.UndefinedBehaviorError
+
+object MhtmlTodo extends JSApp {
+
+  case class Component[D](view: Node, model: Rx[D])
+
+  class Todo(val title: String, val completed: Boolean)
+  object Todo {
+    def apply(title: String, completed: Boolean) = new Todo(title, completed)
+    def unapply(todo: Todo) = Option((todo.title, todo.completed))
+  }
+
+  case class TodoList(text: String, hash: String, items: Rx[Seq[Todo]])
+
+  //
+  // Add components for the app
+  //
+
+  lazy val header: Component[Option[Todo]] = {
+    val newTodo = Var[Option[Todo]](None)
+    val onInputKeydown: (KeyboardEvent) => Unit = { event: KeyboardEvent =>
```

Comment:
That really a style preference, but I think this reads better as a def: (and with `onInputKeydown _` later)

```
def onInputKeydown(event: KeyboardEvent): Unit = {
```